src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -107,6 +114,19 @@ public function iShouldSeeTheProduct(string $name): void
         ));
     }
 
+    /**
+     * @Then I should see that it is out of stock
+     */
+    public function iShouldSeeItIsOutOfStock(): void
+    {
+        /** @var ProductVariantInterface $productVariant */
+        $productVariant = $this->sharedStorage->get('productVariant');
+
+        $variantResponse = $this->productVariantClient->showByIri($this->iriConverter->getIriFromItem($productVariant));
```

Comment:
You can use here `$this->client` instead of adding here another client service
```suggestion
        $variantResponse = $this->client->showByIri($this->iriConverter->getIriFromItem($productVariant));
```